### PR TITLE
Add player ready info to prepared component

### LINKED ISSUE #236: Better ready/unready players information
We previously attempted to add manual start for the room host on #238, but as 
@indocomsoft pointed out the manual state changing caused some dangerous side effects in room  state changes. 

As an alternative, we can just display better information about ready/unready players in the room `PREPARED` state, so you know who to pester to ready up.

Possible extension: host to kick people from the room who may be coming in from a buggy device?

### PR DESCRIPTION
**Pull request by juxd**
Fix #236 

How it looks on the phone:
image

(Activatable by pressing the player count on the prepared screen)

### CODE REVIEW

**Comment by afterdusk** [comment]
Works great for me! Code looks great to me as well, but the rest may have comments so I'll let them approve. Also, I'm not sure if it'll be better to just have the header of the Modal list to be "Players" or "Status".

File: Web/src/components/room/PreparedComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+`;
+
+const NameSpan = styled.span`
+  width: calc(100% - 2em);
+  text-overflow: ellipsis;
+  overflow: hidden;
+`;
+
+const PlayerListContainer = styled(ListItemContainer)`
+  width: 100%;
```
Comment (hzxcaryn) [inline Web/src/components/room/PreparedComponent.tsx]: When we have to many players connected, the list will flex it's way out of the screen hahaha so let's add this 

```suggestion
  width: 100%;
  max-height: 50vh;
  overflow-y: auto;
```

File: Web/src/components/List.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  color: black;
+  padding: 18px 18px;
+  border-top: 1px solid var(--light-grey);
+
+  :first-child {
+    border: none;
+  }
+`;
+
+// separate component so we can have a more divergent styling in future
```
Comment (hzxcaryn) [inline Web/src/components/List.tsx]: @afterdusk is this still happening? HAHAHA

File: Web/src/components/room/PreparedComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+        && <MentalSumsInstructions actions={actions} />}
+    {game === Game.SEQUENCE
+        && <PandaSequenceInstructions actions={actions} />}
+  </>
+);
+
+const NumPlayersContainer = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
```
Comment (hzxcaryn) [inline Web/src/components/room/PreparedComponent.tsx]: To make it more obvious it's clickable
```suggestion
  align-items: center;
  cursor: pointer;
```

File: Web/src/components/room/PreparedComponent.tsx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+`;
+
+const PlayerInfoModal: React.FC<PlayerInfoModalProps> = ({
+  isVisible, allMetas, closeAction,
+}) => (
+  <Modal
+    isVisible={isVisible}
+    title="Player Statuses"
+    confirmationButtonText="Ok"
+    onConfirm={closeAction}
```
Comment (hzxcaryn) [inline Web/src/components/room/PreparedComponent.tsx]:
```suggestion
    onConfirm={closeAction}
    wider
```

**Pull request merged**